src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration/Azure.Configuration.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>This is the implementation of the Azure SDK for Azure Configuration</Description>
-    <AssemblyTitle>Microsoft Azure Configuration</AssemblyTitle>
+    <Description>This is the implementation of the Azure SDK for Azure App Configuration Service</Description>
+    <AssemblyTitle>Microsoft Azure App Configuration</AssemblyTitle>
```

Comment:
I wonder if we should not standardize here. What about something like the following syntax:
Azure SDK <package_name> component
in this case:
Azure SDK Azure.ApplicationModel.Configuration component

@loarabia, @AlexGhiondea?

Reply:
I am ok with that. It will show consistency

Reply:
The official name of the service is `Azure App Configuration` we have ApplicationModel. 
Is there a reason why we make that addition?

Reply:
We will have component for many services under Azure.ApplicationModel. ApplicationModel sounds more like a .NET namespace group, than Azure.App. To me Azure.App sounds like it is the application, not a component for an application. In other words, I tried to make it more idiomatic to how we name namespaces in .NET.

Reply:
Naming will be more clear when other services are under that namespace.
Aggregating other services under `Azure.ApplicationModel` does sound better than `Azure.Application` . It won't match the service naming 1:1 but close enough.